src/lib/openapi/spec/client-features-query-schema.ts

```diff
@@ -4,30 +4,46 @@ export const clientFeaturesQuerySchema = {
     $id: '#/components/schemas/clientFeaturesQuerySchema',
     type: 'object',
     additionalProperties: false,
+    description: 'Query parameters active for a client features request',
     properties: {
         tag: {
             type: 'array',
+            description: 'Features tagged with one of these tags are included',
             items: {
                 type: 'array',
                 items: {
                     type: 'string',
                 },
             },
+            example: [['simple:payment', 'simple:stripejourney']],
```

Comment:
Interesting. Why is it a list of lists here? It's probably how the API is designed, but it seems weird.